float32 a_ide
float32 a_act
float32 d_act
float32 d_bra
float32 v_act

